### PR TITLE
Admin Page : Update static function in get_modules() with use :: operator.

### PR DESCRIPTION
**Pull request by Umangvaghela**
Tested : 

Fix minor changes in class-jetpack-admin.php get_modules().

### CODE REVIEW

**Comment by Umangvaghela** [comment]
@jeherve

Hello sir 

Can you review this PR when you get a leisure time?

Thanks

**Comment by Umangvaghela** [comment]
@oskosk 

This changes is same as #8153
Thanks

**Comment by oskosk** [comment]
@Umangvaghela, please, we need to have "Testing Instructions" in PRs. We need to check that everything still works after the changes in every PR.

Writing Testing Instructions allow us to follow a series of steps to check that no [regression](https://en.wikipedia.org/wiki/Software_regression) is introduced...  Sometimes small changes look great but they end up messing with other pieces.

Writing Testing instructions is easy. For example, if you update a file related to stats, just add something like this to your PR description:

```
#### Testing instructions

* Get to the Jetpack Dashboard
* Confirm that you get no errors in `debug.log` and that the stats chart shows fine
```

Also, you mentioned #8153 in your last comment. That's probably a typo.

I'm labeling this PR as "Needs Author Reply" and blocking it requesting that you add testing instructions before it gets reviewed. I'm really thankful for your contributions to this codebase but we've already mentioned this a few times.

**Comment by Umangvaghela** [comment]
@oskosk 

Thank you, sir, for your response.
I tested all functionality which is related to this function.

#### Testing instructions
Before testing:
* Click Jetpack Dashboard and setting page and also create custom post type: portfolio and taxonomy.
* Create an error.log file and no error found and all are working fine.

After Testing:
* Go to the Jetpack Dashboard and setting page and also check the all custom post type and sharing functionality are working fine.
* Get the same results to change the code.
* Create an error.log file and no error found and all are working fine.

Existing code provides the same result as my changeable code. But I use the scop operator to call a static function.

Thank you, Sir, To conscious me. while I Add PR that time I definitely add testing instruction.

**Comment by Umangvaghela** [comment]
@oskosk 

Greetings of the day.

Can you review this PR when you get a leisure time and give your feedback if I miss something.

Thanks

**Comment by Umangvaghela** [comment]
@oskos , @jeherve 

Sir can you told me what I miss to add in this PR. It is fine or not .

Thanks

File: class.jetpack-admin.php [hunk 1]
```diff
@@ -60,12 +60,13 @@ static function sort_requires_connection_last( $module1, $module2 ) {
 	// presentation like description, name, configuration url, etc.
 	function get_modules() {
 		include_once( JETPACK__PLUGIN_DIR . 'modules/module-info.php' );
-		$available_modules = $this->jetpack->get_available_modules();
-		$active_modules    = $this->jetpack->get_active_modules();
+		$get_object_module = $this->jetpack;
```
Comment (zinigor) [inline class.jetpack-admin.php]: Let's just call the method statically using the class name like it's done everywhere else: `Jetpack::get_available_modules()`. Thank you!

**Comment by Umangvaghela** [comment]
@zinigor 

I have added all changes which is you suggested me. Please let me know if i miss something.

Thanks

**Pull request merged**